### PR TITLE
Created an array that calls api

### PR DESCRIPTION
**Pull request by dvan2**
I added an array that stores the team logo and team name.  We can add create more arrays like this if we want to minimize api calls.  

Basically it will check if the array is null and only if it is null, it will make the API call:
image

image

image

Not sure if it's fully correct so give me feedback.

### CODE REVIEW

File: cs464-group-project/src/components/Api/ApiRequest.js [hunk 1]
```diff
@@ -41,3 +41,27 @@ export async function getAllTeamNames() {
...
+
+export async function getTeamLogos() {
+  //if empty do the call
+  if (allTeamLogos === null) {
+    try {
+      const data = await getAllTeamNames();
+      allTeamLogos = data.teams.map((team) => ({
+        name: team.strTeam,
+        logo: team.strTeamLogo,
+      }));
```
Comment (txiong002) [inline cs464-group-project/src/components/Api/ApiRequest.js]: If just using logos and nothing else, this should work.  But I would include the idTeam prop as well.

**Review by txiong002 — CHANGES_REQUESTED** [review]
It works if you just want to display logos.  For TeamMainPage, i wont be able to use it, because I'm passing the whole team object to the individual page.

File: cs464-group-project/src/components/Api/ApiRequest.js (hunk 1 above)
Comment (dvan2) [reply cs464-group-project/src/components/Api/ApiRequest.js]: I've added idTeam in the allTeamLogos array.  I think when ever we might need to share api calls, we can just create new objects for it.  So we can just write our own stuff like the way we wrote our own API calls.

**Review by txiong002 — APPROVED** [review]
Looks good!

**Pull request merged**